js/pages/samples.js

```diff
@@ -313,26 +314,25 @@ const sanford = {
     name: 'Sanford',
     donatingSamples: 'As part of Connect, we ask you to donate blood, urine, and saliva samples and complete a short survey.',
     whenToDonate: 'The Connect team will send a MyChart message when it is time to donate your samples. If you do not have a MyChart account, we will send you an email. Be sure to check your spam or junk folder. After you receive the MyChart message or email, it is important to donate your samples as soon as you can. It is easy to donate all of your samples in one visit.<br><br><span style="font-weight:900; text-decoration:underline">Note:</span> If you have recently had a blood transfusion or donated blood, please wait at least <span style="font-weight:900; text-decoration:underline">eight weeks</span> from your donation or transfusion before donating your samples for Connect. If you have recently donated plasma, please wait at least <span style="font-weight:900; text-decoration:underline">two days</span> from your plasma donation before donating samples for Connect. If you have an upcoming colonoscopy, please be sure that you <span style="font-weight:900; text-decoration:underline">do not</span> donate samples for Connect on the <span style="font-weight:900; text-decoration:underline">same day</span> as your colonoscopy.',
-    howToDonate: 'The MyChart message or email we send you will contain a link to schedule an appointment. Simply click the link to schedule a time that is convenient for you to donate your samples. Be sure you are scheduling an appointment in the city you are closest to.',
+    howToDonate: 'The MyChart message or email we send you will contain a link to schedule an appointment. Simply click the link to schedule a time that is convenient for you to donate your samples.',
```

Comment:
```suggestion
    howToDonate: 'The MyChart message or email we send you will contain a link to schedule an appointment. Simply click the link to schedule an appointment at a time and location that is convenient for you.',
```
From this doc: https://nih.app.box.com/file/805344518276?s=e5xonzmm27rvd77crt1mnrbalit51a7v is this one current?

Reply:
I reviewed the new doc - please update this per the suggestion.